frontend/src/components/Header/components/Nav/index.tsx

```diff
@@ -24,58 +23,42 @@ export default function Nav({ className, pathname }: Props): JSX.Element {
         <NavSection>
           <NavSectionTitle>Application</NavSectionTitle>
           <NavItemContainer>
-            <LinkWrapper>
-              <Link href={ROUTES.COLLECTIONS} passHref>
-                <AnchorButton
```

Comment:
@seve I removed `AnchorButton` usage since it violates React 18 usage. I think the nav link still works after I updated the style. But PTAL in case I missed anything 🙏 Thank you!